packages/design-system/src/components/Dropdown/DropdownMenuItem.tsx

```diff
@@ -57,9 +59,13 @@ export const DropdownMenuItem = ({
       }
       onMouseEnter={onMouseEnter}
       onClick={(event: React.MouseEvent<HTMLButtonElement>) => {
-        onClick(event);
-        setShown(false);
-        focusManager.focusToggle();
+        // To prevent closing of the dropdown when selecting an item, return DROPDOWN_PREVENT_DEFAULT from the onClick of the item
+        const preventClose =
```

Comment:
I definitely like the concept of making this an explicit option ... but I think it would be easier to follow if it was just passed as a prop instead of requiring a magic return value

Reply:
I can see that being more straightforward. I don't think there's any real use case that a prop wouldn't be able to meet that the return value would.

Reply:
Good idea. Updated.